scripts/build_ros2_workspace.bash

```diff
@@ -6,7 +6,7 @@ if [[ $1 == "-h" ]] || [[ $1 == "--help" ]]; then
   echo -e "Usage: build_ros2_workspace.bash [option...] \t This script builds px4_ros_com workspace for ROS 2" >&2
   echo
   echo -e "\t--add_ros1_bridge \t Clone ros1_bridge. To be used in a further build in case a ROS1 workspace is used. Check 'build_all' or 'build_ros1_bridge' scripts."
-  echo -e "\t--ros_distro \t\t Set ROS 2 distro name (dashing|eloquent|foxy|galactic|rolling). If not set, the script will set the ROS_DISTRO env variable based on the Ubuntu codename"
+  echo -e "\t--ros_distro \t\t Set ROS 2 distro name (dashing|eloquent|foxy|galactic|humble|rolling). If not set, the script will set the ROS_DISTRO env variable based on the Ubuntu codename"
```

Comment:
Can you update this also on the other scripts? Thank you!

Reply:
I've added this in all the build scripts. However, I didn't update the system_setup script because it gets a bit tricky without a supported ROS1 version on Ubuntu 22.04. Do you have some suggestions on how to handle this?